### DISCUSSION TITLE
nextjs theme like Gatsby shadowing?

### DISCUSSION THREAD

**Discussion by web-programmer-here** [vercel/next.js#37882] [Help]
Does nextjs has a feature for theme shadowing like Gatsby? 
https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/shadowing/

**Comment by sanmugam-kathirvel** [comment]
I'm looking for the same.  I'm just wanted to switch from Gatsby to nextjs, Do we have any alternative features in nextjs theme like Gatsby shadowing?
